spec/dummy/config/environments/test.rb

```diff
@@ -10,6 +10,6 @@
   config.action_controller.allow_forgery_protection = false
   config.action_mailer.default_url_options = { host: 'localhost:3000' }
   config.action_mailer.delivery_method = :test
-  config.action_mailer.perform_deliveries = false
+  config.action_mailer.perform_deliveries = true
```

Comment:
In theory, the preview tests ensure that templates compile, and other tests ensure that the delivery tasks are scheduled. So perhaps there is no need for this.